GeckoViewQuickStart.md

```diff
@@ -0,0 +1,132 @@
+
+# GeckoView Quick Start Guide
+
+## Get set up with Mozilla Central
+
+The GeckoView codebase is part of the main Firefox tree and can be found in `mozilla-central`. You will need to get set up as a contributor to Firefox in order to contribute to GeckoView. To get set up, follow the [Quick Start Guide for Git Users](MozCentralQuickStart.md), or the [Contributing to the Mozilla code base](https://developer.mozilla.org/docs/Mozilla/Developer_guide/Introduction) guide on [MDN](https://developer.mozilla.org/) for Mercurial users.
+
+Once you have a copy of `mozilla-central`, you will need to build GeckoView.
+
+## Bootstrap Gecko
+Bootstrap configures everything for GeckoView and Fennec development.
+
+* Ensure you have central checked out. If this is the first time you are doing this, it may take some time.
+
+```bash
+git checkout central/default
+```
+If you are on a mac, you will need to have the Xcode build tools installed. You can do this by either [installing Xcode](https://developer.apple.com/xcode/) or installing only the tools from the command line by running `xcode-select --install` and following the on screen instructions. Use the ` --no-interactive` argument to automatically accept any license agreements.
+
+```bash
+./mach bootstrap [ --no-interactive]
+```
+* Choose option `4. Firefox for Android` for GeckoView development. This will give you a version of Gecko configured for Android that has not bundled the native code into embedded libraries so you can amend the code.
+* Say Y to all configuration options
+* Edit your `mozconfig` - this can be found in the root of your `gecko` repo - or create a file called `mozconfig` if it does not exist. Change it to read as follows:
+
+```bash
+mk_add_options MOZ_OBJDIR=./objdir-frontend
```

Comment:
`objdir-frontend` seems strange to me, maybe `objdir-android-opt`? Also we should encourage folks to separate their objdir from the builddir (like `../objdir-frontend`) -- otherwise editors can get confused and try to index binary files. Additionally, it pollutes `git status` unless you also edit the `.gitignore`.

Reply:
Interesting. This is not mentioned in any of the other guides I followed (maybe because they are not using git). I shall run some tests and see what works...